packages/web/src/new/scenes/Main/Settings/Settings.tsx

```diff
@@ -140,15 +140,15 @@ const Text = styled.div`
 
 export const Settings: FC = observer(() => {
   const viewModel = useViewModel(SettingsViewModel);
+  const rootViewModel = useViewModel(RootViewModel);
```

Comment:
You should't use `ViewModel`s in different parts of app. ViewModel is about logic of 1 page(sometimes component). 

if you need `walletModel` here, just add it to `SettingsViewModel`